### PR TITLE
docs: update usage to match new API

### PR DESCRIPTION
**Pull request by hacdias**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,21 +24,19 @@ const ipfs = require('ipfs-http-client')()
 
 var exampleIp = '89.114.95.36'
```
Comment (lidel) [inline README.md]:
```suggestion
const exampleIp = '89.114.95.36'
```

File: README.md [hunk 2]
```diff
@@ -24,21 +24,19 @@ const ipfs = require('ipfs-http-client')()
...
-geoip.lookupPretty(ipfs, '/ip4/' + exampleIp, (err, result) => {
-  if (err) {
-    console.log('Error: ' + err)
-  } else {
-    console.log('Pretty result: %s', result.formatted)
-  }
-})
+try {
+  const result = await geoip.lookup(ipfs, exampleIp)
+  console.log('Result: ' + JSON.stringify(result, null, 2))
```
Comment (lidel) [inline README.md]: I'd keep example super simple, remove distractions:
```suggestion
  console.log('Result: ', result)
```

**Review by lidel — CHANGES_REQUESTED** [review]
@hacdias looks good, but API section needs to be updates as well to  state that lookup methods now return promises.

ps. I don't see v4.0.0 in https://github.com/ipfs/ipfs-geoip/releases (but it is on NPM),
did you use `npm run release` ?

**Comment by hacdias** [comment]
@lidel oops, I didn't. I just used `npm release` :o I can write the release manually after this is merged.

**Pull request merged**